.jenkins/jenkins_test1.sh

```diff
@@ -9,6 +9,6 @@ source ~/.bashrc
 
 # Test theano core
 PARTS="theano/compat theano/compile theano/d3viz theano/gof theano/misc theano/sandbox/linalg theano/sandbox/tests theano/scalar theano/scan_module theano/sparse theano/tensor theano/tests theano/typed_list"
-THEANO_PARAM="theano --with-timer --timer-top-n 10"
+THEANO_PARAM="${PARTS} --with-timer --timer-top-n 10"
```

Comment:
I don't understand what this PR try to do. Here it don't test theano/sandbox/cuda, so it don't test the old backend.. The comment at the top of this file tell that. The init_gpu_device indicate that.

Do you want test1 to only test the CPU? and test2 to test both GPU back-end?

What this PR try to do?